ckcp/openshift_dev_setup.sh

```diff
@@ -240,9 +245,8 @@ patches:
   echo -n "  - Workspace: "
   # It's not allowed to create WorkloadCluster resource in a non-universal workspace
   if ! KUBECONFIG="$KUBECONFIG_KCP" kubectl get workspaces demo >/dev/null 2>&1; then
```

Comment:
Because there's no else, I believe there will a bug if the `demo` workspace exists, but we're in another workspace at the moment.
1. Make sure we're in the root workspace.
2. Move the `kubectl kcp workspaces use demo` out of the `if`.

Reply:
fixed.